misc/dbt-materialize/setup.py

```diff
@@ -24,7 +24,7 @@
     # This adapter's minor version should match the required dbt-postgres version,
     # but patch versions may differ.
     # If you bump this version, bump it in __version__.py too.
-    version="1.4.1",
+    version="1.5.0",
```

Comment:
We don't want to bump the version just yet, in order to not trigger a release (see [PyPI release step](https://github.com/MaterializeInc/materialize/blob/c1938366ddf784fabe229817420c1b88ceffaf14/ci/deploy/pypi.py#L37-L47)).